codenames-web-services/src/main/java/com/codenames/controllers/AdminWebSocketController.java

```diff
@@ -0,0 +1,72 @@
+package com.codenames.controllers;
+
+import com.codenames.enums.GameStatus;
+import com.codenames.filters.method.GameStoppedFilter;
+import com.codenames.filters.method.UserAuthorizedFilter;
+import com.codenames.models.game.CodeNamesGame;
+import com.codenames.models.game.Player;
+import com.codenames.models.room.Room;
+import com.codenames.models.room.Settings;
+import com.codenames.services.GameService;
+import com.codenames.services.PlayerService;
+import com.codenames.services.RoomService;
+import com.jjerome.annotations.SocketController;
+import com.jjerome.annotations.SocketMapping;
+import com.jjerome.annotations.SocketMappingFilters;
+import com.jjerome.dto.Request;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SocketController
+@RequiredArgsConstructor
+public class AdminWebSocketController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AdminWebSocketController.class);
+
+    private final GameService gameService;
+
+    private final RoomService roomService;
+
+    private final PlayerService playerService;
+
+    private final CodeNamesGame codeNamesGame;
+
+    @SocketMapping(reqPath = "/room/admin/start")
+    @SocketMappingFilters(filters = {
+            UserAuthorizedFilter.class,
+            GameStoppedFilter.class,
+//            UserIsRoomAdminFilter.class
```

Comment:
leave TODO or remove

Reply:
This filter is commented out for testing because without authorization, users do not work adequately, so checking for an administrator is excluded

Reply:
Then leave TODO with this comment :)